Dart/src/com/jetbrains/lang/dart/DartBundle.properties

```diff
@@ -268,4 +268,7 @@ analysis.server.status.bad.text=Analyzer: Submit Feedback (analysis process not
 analysis.server.status.bad.desc=Analysis server needs to be restarted
 
 dart.feedback.url.template=https://github.com/dart-lang/sdk/issues/new?body=Analyzer Feedback from IntelliJ\n\
-  IDEA: {0} Dart: {1} {2}
\ No newline at end of file
+  IDEA: {0} Dart: {1} {2}
+dart.analysis.server.error=Dart analysis server error
+dart.error.file.instructions=Please insert the content of this file here:\n\
```

Comment:
Those temp file URLs are very long, spanning a full line. Putting it on a separate line makes it easy to select and paste into a new browser tab. It seems markdown (or github) only formats links to http URLs, too (these are file: ones), so this isn't even a link.